### PR TITLE
add homework

### PR DESCRIPTION
**Pull request by kaina1129**

### CODE REVIEW

File: day7/kaina/index.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      />
+      <div class="mobile-container">
+        <div class="mobile-logo">
+          <img src="Group 427.png" alt="logo border" />
+          <p>Saturn is best experienced on our mobile app</p>
+        </div>
+        <button>Download Saturn</button>
+      </div>
+    </div>
+    <div class="tablet-wrapper">
```
Comment (mxdi9i7) [inline day7/kaina/index.html]: this wrapper is for both tablet and laptop, don't name it tablet-something, use a more generic name

File: day7/kaina/portrait.html [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (mxdi9i7) [inline day7/kaina/portrait.html]: remove this file if it's not being used

File: day7/kaina/index.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+          </div>
+          <div class="bottom-log-out">
+            <button><i class="fas fa-sign-in-alt"></i>&nbsp;Log Out</button>
+          </div>
+        </div>
+        <div class="tablet-bottom-body">
+          <div class="body-number">
+            <div class="body-number-border text-grey-out">1</div>
+          </div>
+          <div class="body-text text-grey-out">
```
Comment (mxdi9i7) [inline day7/kaina/index.html:65]: your first item is not greyed out

File: day7/kaina/portrait.html (hunk 2 above)
Comment (knazhang) [reply day7/kaina/portrait.html]: removed

File: day7/kaina/index.html (hunk 1 above)
Comment (knazhang) [reply day7/kaina/index.html]: changed name to deskblet

Comment (mxdi9i7) [reply day7/kaina/index.html]: use a more generic name - meaning: do not mention tablet or desktop

**Comment by mxdi9i7** [comment]
remove the .DS_Store file to avoid conflicts

**Pull request closed without merging**